generated/nodejs/google-cloud-node/packages/monitoring/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "url": "https://github.com/googleapis/googleapis",
+  "name": "@google-cloud/monitoring",
+  "version": "0.7.1",
+  "author": "Google Inc.",
+  "description": "Monitoring library for Node.js",
+  "main": "src/index.js",
+  "files": [
+    "src"
+  ],
+  "dependencies": {
+    "arguejs": "^0.2.3",
```

Comment:
Please synchronize your packman to https://github.com/googleapis/packman/pull/138 -- or you could manually edit this package.json for now:
- remove arguejs -- that's not used anymore 
- updagte google-gax dependency to 0.7.0